src/structs.jl

```diff
@@ -228,9 +228,11 @@ JuMP.value(F::Function, P::ScalarParameter) = P.value
 
 function set_value!(P::ScalarParameter, value::Number)
     P.value = value
-    if !isnothing(jump_model(model(P)))
```

Comment:
In an earlier version the jump model wasn't created until solve. This was changed, but residuals remain. I've removed it because it's doing nothing